frontend/src/pages/my-projects/components/create-project-form/libs/types/project-category-to-readable.type.ts

```diff
@@ -0,0 +1,12 @@
+import { ProjectCategory } from '~/packages/projects/projects.js';
```

Comment:
it a map not a type

`frontend/src/pages/my-projects/components/create-project-form/libs/maps/project-category-to-readable.map.ts`